kolibri/deployment/default/settings/base.py

```diff
@@ -202,15 +233,25 @@
 }
```

Comment:
The comments below were a fly-by cleanup to help make the different settings sections clearer.